### ISSUE TITLE
A way to save my main menu items please

### ISSUE THREAD

**Issue by frankee123** [dobbelina/repository.dobbelina#876]
**Is your feature request related to a problem? Please describe.**
A few times I must reinstall cumination as a last resort as features stop working for me or go slow for odd reasons. It's a hassel to remember what I have on my main menu and wondered if there could be a way to save them or back them up as there are for favorites and keywords?

**Describe the solution you'd like**
A way to save my main menu items please. Ty

**If it's a site addition request, add url below**
Sites with no DMCA information will not be added

**Comment by dobbelina** [comment]
By "main menu items" are you referring to Cumination only or addons in general?
What platform are you on?

If you are referring to Cumination only, under windows copy
this folder and restore if needed:
`C:\Users\Your_Name\AppData\Roaming\Kodi\userdata\addon_data\plugin.video.cumination`
Personally I like to set a custom location for the favorites.db file.

**Comment by frankee123** [comment]
I'm sorry if I wasn't clear. I mean cumination only. As if right when you click on it, you see the main menu items like your favorites folder, sites, and that is where I add a lot of my fav sites that I personalize there. I am on an android box.  Without going into detail, I think I am going to have to uninstall cumination and/or kodi and do a fresh reinstall soon. There have been some issues I have had recently and I think instead of posting them on here, trying to get the kodi logs etc., it may be easier to just reinstall lol.

**Comment by imurreflection** [comment]
Have you tried Backup/Restore from Crew Wizard. You can backup your entire build and restore when you need it. You will have exact replica of your Kodi installation including  main menu, any settings including keywords and your favorites both internal and external.

**Comment by frankee123** [comment]
> Have you tried Backup/Restore from Crew Wizard. You can backup your entire build and restore when you need it. You will have exact replica of your Kodi installation including main menu, any settings including keywords and your favorites both internal and external.
> 
> Don't forget to copy the backup file to the external USB drive using ES Explorer.

I have not my friend thank you. I am not savvy when it comes to things on here like you guys haha. I will look into that for sure.

**Comment by frankee123** [comment]
Hey bro, I'm not looking to save my favorite from kodi itself, I wanted to try to save everything in my cumination main menu. I'm not sure if you know what i'm talking about but want to make sure. Are you are saying I can backup and save my cumination main menu items by using the crew wizard, or do you mean save things under favorites from just kodi?

**Comment by frankee123** [comment]
I think this android box having android 11 has been a nightmare. I installed the es explorer and it shows files empty. I had this same damn issue a while ago with somebody trying to help me get into my kodi folder. I think it's something to do with android 11 like it blocks you from seeing all of the folder contents. I don't see or know how I could even back up anything from kodi. Uggghh I'm not sure what to do now really.

**Comment by imurreflection** [comment]
@frankee123 everything..including main menu in Cumination..all folders..your favorites, keywords..just search crew wizard on YouTube. Don't worry it's not that tech. In ES Explorer, you have to select Hidden files..

**Comment by dobbelina** [comment]
You can reach the settings folder with the favorites.db file from inside kodi.
Just go to System and open the File manager, then the path is below:
`Profile directory=>addon_data=>plugin.video.cumination`
From there you can copy it.
Make sure you dont forget to "Add source" first so you have somewhere
to copy it.

**Comment by frankee123** [comment]
Ty both for the responses and help, but I have hidden files selected and it's still empty. I know somebody told me it's because of android 11 they don't let you view any files. This was a painful process to figure out a while back when I posted on here unfortunately. I cannot get into any of my files on my android box...Ty dobb, I found what you were talking about. I have copied the files advanced settings.xml, cookies.lwp, favorites.db, and settings.xml. I pasted them to my external usb just now. Can you tell me please if the favorites.db file is the cumination main menu items I have? I hope that is it, that is really what I am looking for. Ty both again.

**Comment by frankee123** [comment]
I'm pretty sure with your help, that the favorites.db may not be what I am looking for unfortunately. i may have to try to post the errors that I have been getting instead. It's two issues I have been having. I don't want to put them in here, I know I got to make a new post with my error log etc.  I will say that it's that nothing will play in 4k using the program Spankbang for some reason. The other one is half the time I click to play a online chaturbate girl, it tries to load and play the stream, but after about a minute, it just says that playback has failed and then I have to click on the girl again and it will open. It just happens so constantly, that it has become a major nuisance and I want to fix it.

**Comment by frankee123** [comment]
Ugh the kodi log file uploader is saying installation failed when I try to install now for some reason?

**Comment by DJMix-I-Can** [comment]
Honestly, since you're constantly having problems with your installation or setup. You're also probably re installing all of the other addons & re-configuring them. That is time consuming. I don't know how frequently your setup gives you issues but your best bet would be to back up your .kodi folder about once a week. That way you won't have to re-install the addons, reconfigure & the sites that you saved on Cuminations main menu will also still be there. I personally do that. I could do it once a week, twice a week or once a month. It all depends if there has been a lot of updates or changes on my behalf. If anything fails, I just extract that zip file and I'm back to my previous setup without having to reconfigure or reauthorize anything. I believe I spoke to you about this a while back with your Android 11 permission issues and suggested to use CX File Explorer in order to view your .kodi folder. Did you use it?

**Comment by frankee123** [comment]
Yes, DJ, thank you for the response and helpful tips. I tried that cx file explorer and any other one you guys have suggested. NOTHING works bro. All the files are hidden or it'll give me a error saying the program isn't compatible with my android box, stuff like that. I make sure to give the app permission and to check show hidden files etc but nothing shows my files. I am going to upload my kodi logs now since the kodi logfile uploader finally installed so hopefully somebody can figure out what is wrong. Ty

**Comment by DJMix-I-Can** [comment]
I just noticed this comment you posted to Dobbelina, "Ty dobb, I found what you were talking about. I have copied the files advanced settings.xml, cookies.lwp, favorites.db, and settings.xml. I pasted them to my external usb just now. " How did you do that if those files are in the .kodi folder which is supposedly hidden & can't view it??

**Comment by frankee123** [comment]
Oh I meant using the cx file explorer and any program that is made to view files on my android box is all. What Dobb posted was the only thing I was able to find since it is actually inside of Kodi I am able to find those files he talked about. I hope you understand what I mean bro.

**Comment by frankee123** [comment]
I made a different post with hopefully the correct info in the kodi log file if interested. Thx

**Comment by DJMix-I-Can** [comment]
No I don't because technically you shouldn't have been able to view those files. Those files are within the .kodi folder which you stated you can't view. Once you click on the .kodi folder, you should've received a permission denied warning. No folders should've showed up.

**Comment by frankee123** [comment]
Hmm I'm not sure what to tell you bro. I can view the files what dobb posted about within Kodi itself, but outside of it and trying to use the programs like the cx file explorer etc...none of those will work.
